runbotics/runbotics-orchestrator-ui/src/components/utils/Secured.tsx

```diff
@@ -23,6 +23,20 @@ export const hasAuthorities = (user: User, authorities: any[]) => {
     return true;
 };
 
+export const hasAdminAuthority = (user: User, roles: Role) => {
+    if (!user || !user.roles) {
+        return false;
+    }
+
+    for (const role of roles) {
```

Comment:
you are iterating over a single enum, don't know how eslint didn't highlighted it